src/travis2docker/templates/Dockerfile_deployv

```diff
@@ -17,7 +17,7 @@ RUN {% for src, dest in copies or [] -%} chown -R {{ user }}:{{ user }} {{dest}}
 RUN cat ${HOME}/.ssh/id_rsa.pub | tee -a ${HOME}/.ssh/authorized_keys
 {%- endif %}
 
-ENV ODOORC_DB_NAME=odoo ODOORC_PIDFILE=/home/odoo/.odoo.pid ODOORC_DATA_DIR=/home/odoo/data_dir PROFILE_FILE=/etc/bash.bashrc MAIN_REPO_FULL_PATH=/home/odoo/instance/$MAIN_REPO_PATH DEB_PYTHON_INSTALL_LAYOUT=deb
+ENV ODOORC_DB_NAME=odoo ODOORC_PIDFILE=/home/odoo/.odoo.pid ODOORC_DATA_DIR=/home/odoo/data_dir PROFILE_FILE=/etc/bash.bashrc MAIN_REPO_FULL_PATH=/home/odoo/instance/$MAIN_REPO_PATH DEB_PYTHON_INSTALL_LAYOUT=deb PRE_COMMIT_HOME=/usr/local/lib/python3.8/dist-packages/pre_commit_vauxoo
```

Comment:
@moylop260 Do you know of a better place where I could place `PRE_COMMIT_HOME`. Ideally I don't want to hardcore it, especially since it can easily be set with `export PRE_COMMIT_HOME=$(python3 -c 'import pre_commit_vauxoo; print(pre_commit_vauxoo.__path__[0])')`

Reply:
If I understood well Dockerfile ENV is not able to eval `$()` neither vscode

You could use the default HOME paths of pylint and flake8

I mean, you can copy the configuration file to ~/.flake8 and ~/.pylintrc directly and change the owner to $USER

So, it will be use by default since vscode without extra parameters